client/my-sites/advertising/useAdvertisingUrl.jsx

```diff
@@ -0,0 +1,19 @@
+import { isEnabled } from '@automattic/calypso-config';
+import { useSelector } from 'react-redux';
+import { getSiteAdminUrl, getSiteOption } from 'calypso/state/sites/selectors';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+
+const useAdvertisingUrl = () => {
+	const selectedSiteSlug = useSelector( getSelectedSiteSlug );
+	const siteId = useSelector( ( state ) => getSelectedSiteId( state ) );
+	const siteAdminUrl = useSelector( ( state ) => getSiteAdminUrl( state, siteId ) );
```

Comment:
Feel free to ignore and this might be intentional, but `getSiteAdminUrl` accepts the path as a third parameter, so you could do something like this if you'd prefer:

```suggestion
	const siteAdminUrl = useSelector( ( state ) => getSiteAdminUrl( state, siteId, 'tools.php?page=advertising' ) );
```

Reply:
Oh, nice! I added it. Thanks!